app/models/statutory_instrument.rb

```diff
@@ -27,7 +27,7 @@ def self.title
 
   def links
     super.merge(
-      organisations: organisations,
+      organisations: organisations << primary_publishing_organisation,
```

Comment:
This will result in duplicates if they put the org in both fields, probably want `organisations | [primary_publishing_organisation]`